client-sdk/ts-web-ledger/playground/dist/index.html

```diff
@@ -0,0 +1,4 @@
+<!doctype html>
```

Comment:
Should this playground be integrated as some kind of a test case? Probably not in this PR, as we need to add tests all around the SDK anyway.

Reply:
should be. it'll be more feasible after I switch over to oasis-net-runner instead of running on testnet